server/routes/friends/friend.js

```diff
@@ -54,6 +54,21 @@ router.post('/friends', [authentication], async function (req, res) {
   }
 });
 
+//@route            GET /api/friends/:friend_id
+//@desc             get a friend info by id
+//@access           Private
+router.get('/friends/:friend_id', [authentication], async (req, res) => {
```

Comment:
why do you wrap auth middleware in array?? are you expecting to add some more middleware ??